packages/eslint-plugin-pf-codemods/src/rules/v6/labelRemoveIsOverflowLabel/label-remove-isOverflowLabel.test.ts

```diff
@@ -0,0 +1,24 @@
+const ruleTester = require('../../ruletester');
+import * as rule from './label-remove-isOverflowLabel';
+
+ruleTester.run("label-remove-isOverflowLabel", rule, {
+  valid: [
+    {
+      code: `<Label isOverflowLabel />`
+    },
+    {
+      code: `import { Label } from '@patternfly/react-core'; <Label someOtherProp />`
+    }
+  ],
+  invalid: [
```

Comment:
This will require an update to be made either to the renameProps helper or this rule to be more custom, but we should add an invalid test that checks whether an existing `variant` prop is replaced or not:

```javascript
{
  code:   `import { Label } from '@patternfly/react-core'; <Label isOverflowLabel variant="outline" />`,
  output: `import { Label } from '@patternfly/react-core'; <Label variant="overflow" />`,
  errors: [{
      message: `isOverflowLabel prop for Label has been replaced with variant="overflow"`,
      type: "JSXOpeningElement",
      }]
    },
```

I'd be fine having this be a followup, though, especially since we can't be sure if consumers would be passing both isOverflowLabel and a variant (since the variant shouldn't have any effect for an overflow label I believe).

Reply:
Good point! I think we can do the change right now, I will make this rule more custom